html/elements/time.json

```diff
@@ -0,0 +1,131 @@
+{
+  "html": {
+    "elements": {
+      "time": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/time",
+          "support": {
+            "webview_android": {
+              "version_added": "62"
+            },
+            "chrome": {
+              "version_added": "62"
+            },
+            "chrome_android": {
+              "version_added": "62"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22"
+            },
+            "firefox_android": {
+              "version_added": "22"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": [
+              {
+                "version_added": "11.5",
+                "version_removed": "12"
+              },
+              {
+                "version_added": "49"
+              }
+            ],
+            "opera_android": [
+              {
+                "version_added": "11.5",
+                "version_removed": "12"
+              },
+              {
+                "version_added": "49"
+              }
+            ],
+            "safari": {
+              "version_added": "7"
+            },
+            "safari_ios": {
+              "version_added": "4"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "datetime": {
+          "__compat": {
+            "support": {
+              "webview_android": {
+                "version_added": "62"
+              },
+              "chrome": {
+                "version_added": "62"
+              },
+              "chrome_android": {
+                "version_added": "62"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "22"
+              },
+              "firefox_android": {
+                "version_added": "22"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "opera": [
```

Comment:
Same here, less interesting version range after the current one

Reply:
Fixed